features/step_definitions/helper_services.rb

```diff
@@ -153,27 +153,28 @@
   step %Q/evaluation of `pod` is stored in the :proxy_pod clipboard/
 end
 
-Given /^I have LDAP service in my project$/ do
-    ###
-    # The original idea is trying to put ldap server in the openshift to make this flexy.
-    # Since we run the scenario in jenkins slave which is not in sdn, then two choices come to me:
-    # 1, Create a route for the ldapserver pod, but blocked by this us https://trello.com/c/9TXvMeS2 is done.
-    # 2, Port forward the ldap server pod to the jenkins slave.
-    # So take the second one since this one can be implemented currently
-    ###
+###
+# The original idea is trying to put ldap server in the openshift to make this flexy.
+# Since we run the scenario in jenkins slave which is not in sdn, then two choices come to me:
+# 1, Create a route for the ldapserver pod, but blocked by this us https://trello.com/c/9TXvMeS2 is done.
+# 2, Port forward the ldap server pod to the jenkins slave.
+# So take the second one since this one can be implemented currently
+###
+Given /^I have LDAP service in my project$/ do 
     step %Q/I run the :run client command with:/, table(%{
       | name  |ldapserver                             |
       | image |openshift/openldap-2441-centos7:latest |
```

Comment:
In https://github.com/openshift/verification-tests/pull/1353#discussion_r454987617 you said changes among versions. You can check oc run -h to use -l to add your label independent of changes.

Reply:
Wouldn't you like to add your own label here and the following `a pod becomes ready with labels` step to make it robust regardless of versions?

Reply:
As I am not using RC, there is no change in the labels. `run=ldapserver` label is available in the <4.5 and >4.5 version. So its checking for every version. so I think no need to add my own label here.